bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/R4AvroConverterCustomProfileTest.java

```diff
@@ -54,11 +54,15 @@ public static void setUp() throws URISyntaxException, ProfileException {
     AvroConverter converterBunsenTestProfilePatient =
         AvroConverter.forResources(fhirContext, patientProfiles, 1);
 
-    avroBunsenTestProfilePatient =
+    Record testBunsenTestProfilePatientRecord =
```

Comment:
nit: What is the benefit of the new aux variable `testBunsenTestProfilePatientRecord`? If none, then revert it?

Reply:
The other change is reducing the scope the variable from field to local for the specific test case using it.